apps/design/backend/src/test_decks.test.ts

```diff
@@ -22,7 +22,7 @@ import {
 } from './test_decks';
 
 vi.setConfig({
-  testTimeout: 30000,
+  testTimeout: 60_000,
```

Comment:
Maybe there was a perf hit as part of the upgrade, because I had to bump these timeouts substantially to make these pass.

Reply:
I remember something to this effect with the upgrade I did as well. I see there are others having issues as well, and one of the sources of the performance regression was fixed in https://github.com/vitest-dev/vitest/pull/7291. However, that was fixed in v3.0.3. Whether there are other perf issues lurking, I don't know.